pseudomonas_analysis/2_identify_generic_genes_pathways_pao1.ipynb

```diff
@@ -0,0 +1,1764 @@
+    "    sample_id_colname,\n",
+    "    NN_architecture,\n",
+    "    dataset_name,\n",
+    "    scaler,\n",
+    "    local_dir,\n",
+    "    base_dir,\n",
+    "    run,\n",
+    "):\n",
+    "    \"\"\"\n",
+    "    Generate new simulated experiment using the selected_experiment_id as a template\n",
+    "    experiment using the same workflow as `simulate_by_latent_transform`\n",
+    "\n",
+    "    This will return a file with a single simulated experiment following the workflow mentioned.\n",
+    "    This function can be run multiple times to generate multiple simulated experiments from a\n",
+    "    single selected_experiment_id.\n",
+    "\n",
+    "    Arguments\n",
+    "    ----------\n",
+    "    normalized_data_file: str\n",
+    "        File containing normalized gene expression data\n",
+    "\n",
+    "        ------------------------------| PA0001 | PA0002 |...\n",
+    "        05_PA14000-4-2_5-10-07_S2.CEL | 0.8533 | 0.7252 |...\n",
+    "        54375-4-05.CEL                | 0.7789 | 0.7678 |...\n",
+    "        ...                           | ...    | ...    |...\n",
+    "\n",
+    "    selected_experiment_id: str\n",
+    "        Experiment id selected as template\n",
+    "\n",
+    "    sample_id_colname: str\n",
+    "        Column header that contains sample id that maps expression data and metadata\n",
+    "\n",
+    "    NN_architecture: str\n",
+    "        Name of neural network architecture to use.\n",
+    "        Format 'NN_<intermediate layer>_<latent layer>'\n",
+    "\n",
+    "    dataset_name: str\n",
+    "        Name for analysis directory. Either \"Human\" or \"Pseudomonas\"\n",
+    "\n",
+    "    scaler: minmax model\n",
+    "        Model used to transform data into a different range\n",
+    "\n",
+    "    local_dir: str\n",
+    "        Parent directory on local machine to store intermediate results\n",
+    "\n",
+    "    base_dir: str\n",
+    "        Root directory containing analysis subdirectories\n",
+    "\n",
+    "    run: int\n",
+    "        Simulation run\n",
+    "\n",
+    "    Returns\n",
+    "    --------\n",
+    "    simulated_data_file: str\n",
+    "        File containing simulated gene expression data\n",
+    "\n",
+    "    \"\"\"\n",
+    "\n",
+    "    # Files\n",
+    "    NN_dir = os.path.join(base_dir, dataset_name, \"models\", NN_architecture)\n",
+    "    latent_dim = NN_architecture.split(\"_\")[-2] # Here we manually changed the index to work\n",
+    "\n",
+    "    model_encoder_file = glob.glob(os.path.join(NN_dir, \"*_encoder_model.h5\"))[0]\n",
+    "\n",
+    "    weights_encoder_file = glob.glob(os.path.join(NN_dir, \"*_encoder_weights.h5\"))[0]\n",
+    "\n",
+    "    model_decoder_file = glob.glob(os.path.join(NN_dir, \"*_decoder_model.h5\"))[0]\n",
+    "\n",
+    "    weights_decoder_file = glob.glob(os.path.join(NN_dir, \"*_decoder_weights.h5\"))[0]\n",
+    "    \n",
+    "    #print(\"model files used\")\n",
```

Comment:
You can probably remove the commented out print statements in this cell

Reply:
Good catch. Just some leftover debugging stuff